tests/test_discretisation/test_arbitrary_discretiser.py

```diff
@@ -1,12 +1,11 @@
 import numpy as np
 import pandas as pd
 import pytest
+from feature_engine.discretisation import ArbitraryDiscretiser
```

Comment:
pls use isort to tidy the imports. According to PEP imports from own module go at the end.

Reply:
Found the cause:

It depends in which order the virtual environment was setup when using PyCharm. Details see [here](https://youtrack.jetbrains.com/issue/PY-23542/Incorrect-import-sorting-Local-libraries-are-treated-as-third-party)

Some research ended up in following:

Creating a config-file for isort named .isort.cfg with following content:

```
[settings]
profile=black
```

Now isort works as expected and black does not reorder the imports (which happened here as well). Also the build-in sorting function from PyCharm works like isort.